### PR TITLE
Trading page charts v2

### LINKED ISSUE #2607: Integrate V2 getters data for market trading charts
Candlestick chart uses Markets.ts `getMarketPriceCandlesticks` getter

Data is already being pulled, needs to be verified
 * Price History uses market trading history
    - all markets are now categorical so price history will have multiple lines
 * depth chart uses order book

### PR DESCRIPTION
**Pull request by andrewdamelio**
closes #2607 

Update trading page charts
- Align outcome colors with price history chart
- Use new candleStick sdk call
- misc chart fixes/ updates

### CODE REVIEW

**Comment by bthaile** [comment]
Looks like tests are failing. Didnt think to run the Markets.test.ts file. It takes many minites to complete. @andrewdamelio 
It can be run with ‘yarn test (filepath)’

**Comment by andrewdamelio** [comment]
@bthaile fixing tests

File: packages/augur-ui/src/store/select-state.ts [hunk 1]
```diff
@@ -34,14 +34,23 @@ export const selectAccountShareBalance = (state: AppState) =>
 export const selectFilledOrders = (state: AppState) => state.filledOrders;
 export const selectUserMarketOpenOrders = (state: AppState) => state.userOpenOrders;
 
+
+// TODO use Date.NOW until this is wired up
 export const selectCurrentTimestamp = createSelector(
```
Comment (bthaile) [inline packages/augur-ui/src/store/select-state.ts]: blockchain state, currentAugurTimestamp is wired up on master

**Review by bthaile — APPROVED** [review]
lgtm, go ahead and wire up blockchain timestamp in this PR or in additional PR

File: packages/augur-ui/src/store/select-state.ts (hunk 1 above)
Comment (andrewdamelio) [reply packages/augur-ui/src/store/select-state.ts]: wicked -- i'll update this on this PR

**Pull request merged**